app/services/display_ads/sample_relevant_ads.rb

```diff
@@ -0,0 +1,79 @@
+module DisplayAds
+  class SampleRelevantAds
+    def self.call(...)
+      new(...).call
+    end
+
+    def initialize(display_ads:, area:, user_signed_in:, article_tags:)
+      @filtered_display_ads = display_ads
+      @area = area
+      @user_signed_in = user_signed_in
+      @article_tags = article_tags
+    end
+
+    def call
+      @filtered_display_ads = approved_and_published_ads
```

Comment:
@lightalloy I've thought a bit more about this and put out a [poll](https://forem-team.slack.com/archives/C7GE84DEJ/p1674639502333729) about it because I was feeling very uncertain about which way to go. I'd prefer to keep it as is as I think it the naming of the methods provides clarity and make it easy to follow and it has a consistent level of abstraction. Thank you for bringing this up, I appreciate it as it allowed me to think more deeply about it.